docs/reference/frozen-indices.asciidoc

```diff
@@ -29,6 +29,17 @@ data structures on demand which can cause page faults and garbage collections, w
 
 Since indices that are eligible for freezing are unlikely to change in the future, disk space can be optimized as described in <<tune-for-disk-usage>>.
 
+It's highly recommended to <<indices-forcemerge,`_forcemerge`>> you indices prior to freezing them to ensure the indices only have a single
+segment on disk. This will provide much better compression but also prevents the need for additional data-structures required for
```

Comment:
```suggestion
segment on disk. This will not only provide much better compression but also simplify the data structures needed to service aggregation requests.
```

Reply:
You might also consider removing the future tense here.  For example: "This not only provides much better compression but also simplifies..."

Reply:
++